contracts/core/OpLightClient.sol

```diff
@@ -40,11 +39,7 @@ contract OptimisticConsensusStateManager is ConsensusStateManager {
             // if this is a new apphash we need to verify the provided proof. This method will revert in case
             // of invalid proof.
             verifier.verifyStateUpdate(
-                l1header,
-                proof,
-                bytes32(appHash),
-                l1BlockProvider.hash(),
-                l1BlockProvider.number()
+                l1header, proof, bytes32(appHash), l1BlockProvider.hash(), l1BlockProvider.number()
```

Comment:
There's a `.prettierrc.yml` config in the root directory that specifies minor formatting config that we all follow. You may wanna add that to your IDE to prevent changes like this.